content/Google is a modal dialog - Martin Rubi.md

```diff
@@ -0,0 +1,116 @@
+## Google is a modal dialog
+
+### The shame of being a modal dialog
+
+A modal dialog is a window which pops up on top of every other window freezing the
+underlaying user interface until the user dismisses it.
+
+Its use was quite frequent in the early days of desktop windows apps but with the
+beginning of browser user intefaces they not only became obsolete but some articles
+were writen discouraging its use in favor of edit in place interfaces.
+
+I'm no UX expert but I am a user of graphics user intefaces on a daily basis and
+ as a user I will speak up in defense of modal dialogs when used properly.
+
+### The importance of being modal
+
+What do modal dialogs do?
+
+When I first thought in modal dialogs I focused mainly on the space location
+in the user interface.
+
+The first desktop apps achieved 3D emulation by overlapping 2D rectangles called
+windows [1].
+
+Browser interfaces avoid 3D emulation and usually stick to a strict 2D interface
+through laying out its elements instead of using overlapping.
+
+However this is just the implementation of what modal dialogs do and not its functional
+purpose and value.
+
+Modal dialogs interrupt the flow of the user attention to solve a particular task
+providing a self contained, highly focused functional and visual context for the user.
+
+That is they provide for a very short period of time a user interface for the user
+to solve a specific task hidding every other complexity not related to that task.
```

Comment:
This is the most amazing correction I've read since I went to an English class for the last time.
It's so good that it makes want to write an article about learning and making corrections.